tests/unit/aegra/test_mcp_token_store.py

```
@@ -130,22 +130,28 @@ async def test_get_client_returns_none_on_secret_decryption_failure(
             "updated_at": None,
         }
 
+        mock_cur = AsyncMock()
+        mock_cur.fetchone.return_value = fake_row
+        mock_conn = AsyncMock()
+        mock_conn.execute.return_value = mock_cur
+
+        from contextlib import asynccontextmanager
+
+        @asynccontextmanager
+        async def _fake_connection(**kwargs):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_fake_connection`

(ANN202)